sdk/src/test/java/io/opentelemetry/sdk/trace/TracerSdkTest.java

```diff
@@ -106,4 +110,78 @@ public void propagatesInstrumentationLibraryInfoToSpan() {
     ReadableSpan readableSpan = (ReadableSpan) tracer.spanBuilder("spanName").startSpan();
     assertThat(readableSpan.getInstrumentationLibraryInfo()).isEqualTo(instrumentationLibraryInfo);
   }
+
+  @Test
+  public void stressTest() {
+    CountingSpanProcessor spanProcessor = new CountingSpanProcessor();
+    TracerSdkProvider tracerSdkProvider = TracerSdkProvider.builder().build();
+    tracerSdkProvider.addSpanProcessor(spanProcessor);
```

Comment:
I would add the batchsp as well and count the num spans in the exporter. Since it is most likely that batchsp will be almost everywhere.

Reply:
yep. easy to add; i'll make it happen

Reply:
done, as an additional stress test.